### PR TITLE
convertion decimal to binary

### LINKED ISSUE #3846: Convert decimal to binary
### Description

Write a program to convert decimal to binary

Binary is a base-2 number system. It uses only two digits: 0 and 1. The digits are called bits. A bit is the smallest unit of information in a computer. A bit can be either 0 or 1. A bit can be used to represent a single value, such as 0 or 1, or it can be used to represent a single state, such as on or off.

To convert a decimal number to binary, we can use the following algorithm:

1. Divide the decimal number by 2.
2. Write down the remainder.
3. Divide the quotient by 2.
4. Write down the remainder.
5. Repeat steps 3 and 4 until the quotient is 0.
6. The binary number is the sequence of remainders written in reverse order.

```
Input  : 10
Output : 1010
```

### Tracking Issues

- [ ] Write a C program to convert decimal to binary
- [ ] Write a C# program to convert decimal to binary
- [ ] Write a C++ program to convert decimal to binary
- [ ] Write a Dart program to convert decimal to binary
- [ ] Write a F# program to convert decimal to binary
- [ ] Write a GO program to convert decimal to binary
- [ ] Write a Haskell program to convert decimal to binary
- [ ] Write a Java program to convert decimal to binary
- [ ] Write a JavaScript program to convert decimal to binary
- [ ] Write a Julia program to convert decimal to binary
- [ ] Write a Kotlin program to convert decimal to binary
- [ ] Write a Perl program to convert decimal to binary
- [ ] Write a PHP program to convert decimal to binary
- [ ] Write a Python program to convert decimal to binary
- [ ] Write a R program to convert decimal to binary
- [ ] Write a Ruby program to convert decimal to binary
- [ ] Write a Rust program to convert decimal to binary
- [ ] Write a Scala program to convert decimal to binary
- [ ] Write a Swift program to convert decimal to binary
- [ ] Write a TypeScript program to convert decimal to binary

### LINKED ISSUE #3854: Write a Java program to convert decimal to binary
### Description

Write a Java program to convert decimal to binary

Binary is a base-2 number system. It uses only two digits: 0 and 1. The digits are called bits. A bit is the smallest unit of information in a computer. A bit can be either 0 or 1. A bit can be used to represent a single value, such as 0 or 1, or it can be used to represent a single state, such as on or off.

To convert a decimal number to binary, we can use the following algorithm:

1. Divide the decimal number by 2.
2. Write down the remainder.
3. Divide the quotient by 2.
4. Write down the remainder.
5. Repeat steps 3 and 4 until the quotient is 0.
6. The binary number is the sequence of remainders written in reverse order.

```
Input  : 10
Output : 1010
```
<details>
<summary>How to contribute</summary>

- Save the solution in `program/ConvertDecimalToBinary.java` file
- Add `ConvertDecimalToBinary.java` file in `convert-decimal-to-binary` folder

</details>

### PR DESCRIPTION
**Pull request by Emilia-mazari**
### Why:
Closes codinasion/codinasion#2833

### CODE REVIEW

**Review by anandfresh — CHANGES_REQUESTED** [review]
@Emilia-mazari  , Please assign this issue ( https://github.com/codinasion/program/issues/3846) to your name before raising a PR

File: program/convert-decimal-to-binary/ConversationToBinAndDec.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ */
+public class ConversationToBinAndDec {
+
+	/**
+	 * @param args the command line arguments
+	 */
+//	public static void main(String[] args) {
+//		// TODO code application logic here
+//		// System.out.println(dectoHex(16));
+//
```
Comment (anandfresh) [inline program/convert-decimal-to-binary/ConversationToBinAndDec.java:20]: remove the unwanted code comments

**Review by anandfresh — CHANGES_REQUESTED** [review]

**Review by hi-Kartik2004 — COMMENTED** [review]
Hi @Emilia-mazari ✋🏻 , Thanks for linking the issue and making a Pull request (PR) but I'm **getting an error when I try to compile your file.**
The same is attached to this comment, can you resolve this? 
, and I shall approve the changes.😄 
image

**Review by hi-Kartik2004 — APPROVED** [review]

**Review by harshraj8843 — APPROVED** [review]
Thanks for contributing 👍🏻

**Pull request merged**